cli/command/formatter/container.go

```diff
@@ -189,11 +189,16 @@ func (c *ContainerContext) Command() string {
 	return strconv.Quote(command)
 }
 
-// CreatedAt returns the "Created" date/time of the container as a unix timestamp.
+// CreatedAt returns the "Created" date/time of the container as a stringified unix timestamp.
 func (c *ContainerContext) CreatedAt() string {
 	return time.Unix(c.c.Created, 0).String()
 }
 
+// CreatedAtTimestamp returns the "Created" date/time of the container as a unix timestamp.
+func (c *ContainerContext) CreatedAtTimestamp() time.Time {
+	return time.Unix(c.c.Created, 0)
```

Comment:
I wonder if this should just return `c.Created` as-is (and perhaps the function be named `Created()`, and probably needs an entry in the "headers" (when used for table-view). Do we need to look at other formatters?

I'm slightly considering if we should have an option to expose the Container itself for formatting (have ` .Container()` function that returns `c.c`)

Reply:
Good point. What benefit would there be to simply returning the `Container` directly as opposed to presenting these APIs? That feels like a potential leaky abstraction to me, but that's just a gut feeling.

I could definitely see these as redundant since the `Container` does have `Created` accessible, but having other modules do the formatting seems like we could end up with a lot of duplication unless we had an interface to standardize that. Thoughts?